[2024-03-01T16:35:52.335+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: datapipeline_Take4.zipallfiles_and_placeatrequiredlocation scheduled__2024-03-01T11:00:00+00:00 [queued]>
[2024-03-01T16:35:52.355+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: datapipeline_Take4.zipallfiles_and_placeatrequiredlocation scheduled__2024-03-01T11:00:00+00:00 [queued]>
[2024-03-01T16:35:52.355+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-01T16:35:52.453+0530] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): zipallfiles_and_placeatrequiredlocation> on 2024-03-01 11:00:00+00:00
[2024-03-01T16:35:52.463+0530] {standard_task_runner.py:60} INFO - Started process 23004 to run task
[2024-03-01T16:35:52.472+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'datapipeline_Take4', 'zipallfiles_and_placeatrequiredlocation', 'scheduled__2024-03-01T11:00:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/datapipeline_best(2).py', '--cfg-path', '/tmp/tmpwrb5zr8k']
[2024-03-01T16:35:52.476+0530] {standard_task_runner.py:88} INFO - Job 130: Subtask zipallfiles_and_placeatrequiredlocation
[2024-03-01T16:35:52.668+0530] {task_command.py:423} INFO - Running <TaskInstance: datapipeline_Take4.zipallfiles_and_placeatrequiredlocation scheduled__2024-03-01T11:00:00+00:00 [running]> on host arun-VM
[2024-03-01T16:35:53.094+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='datapipeline_Take4' AIRFLOW_CTX_TASK_ID='zipallfiles_and_placeatrequiredlocation' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T11:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T11:00:00+00:00'
[2024-03-01T16:35:53.115+0530] {python.py:201} INFO - Done. Returned value was: None
[2024-03-01T16:35:53.129+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=datapipeline_Take4, task_id=zipallfiles_and_placeatrequiredlocation, execution_date=20240301T110000, start_date=20240301T110552, end_date=20240301T110553
[2024-03-01T16:35:53.315+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-01T16:35:53.548+0530] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
